internal/rbd/controllerserver.go

```diff
@@ -341,6 +349,11 @@ func (cs *ControllerServer) CreateVolume(
 		return nil, err
 	}
 
+	err = updateTopologyConstraints(rbdVol, rbdSnap)
+	if err != nil {
+		return nil, err
```

Comment:
This will return non-grpc error to the RPC, we should return error in GRPC format

Reply:
Added `codes.Internal,` lemme know if something else we can add...